tests/opentelemetry/test_span.py

```diff
@@ -15,6 +15,7 @@
 from tests.utils import flaky
 
 
+@flaky(1735812000)
 @pytest.mark.snapshot
 def test_otel_span_attributes(oteltracer):
```

Comment:
I ran this this a few times locally and could not get it to fail. Can you send me a link to the failure in circle ci

Reply:
https://app.datadoghq.com/ci/test-runs?query=test_level%3Atest%20%40test.status%3Afail%20%40test.name%3Atest_otel_span_attributes&index=citest&start=1704657963597&end=1704744363597&paused=false